packages/block-editor/src/store/selectors.js

```diff
@@ -1079,7 +1079,7 @@ export function isBlockBeingDragged( state, clientId ) {
 export function isAncestorBeingDragged( state, clientId ) {
 	// Return early if no blocks are being dragged rather than
 	// the more expensive check for parents.
-	if ( ! state.draggedBlocks ) {
+	if ( ! state.draggedBlocks.length ) {
```

Comment:
Nitpick: to unify the logic a little bit
```suggestion
	if ( ! isDraggingBlocks(state) ) {
```

Reply:
Nice suggestion 👍